### DISCUSSION TITLE
Error remote: You are not authorized to access this collection.

### DISCUSSION THREAD

**Discussion by Jamesf86** [argoproj/argo-cd#9682] [Q&A] [answered]
Okay, this is an existing installation that was known to be working.  However, once I go in to the argo/argocd web interface and access the apps tab.  I am getting the following error message:
------------------------------------------------
Unable to load data: Failed to fetch default: `git fetch origin --tags --force` failed exit status 128: remote: You are not authorized to access this collection. fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists.
------------------------------------------------
Looking at the setting/repository tab, I see the repository connection to ssh.dev.azure.com has a status of failed.  
So I am not sure where the issue is at, is it on ssh.dev.azure.com or is it on my existing Argo/Argocd installation.

**Comment by 34fathombelow** [comment, marked as answer]
By any chance have you recently upgraded to Argo CD 2.4?  If so, in 2.4 the base image was upgraded to Ubuntu 22.04.  SHA1 ssh signature algorithms are no longer supported by default.   This is something that needs to be upgraded on [ssh.dev.azure.com](ssh.dev.azure.com) side. 

For the workaround and more explanation check `https://argo-cd.readthedocs.io/en/latest/operator-manual/upgrading/2.3-2.4/#support-for-private-repo-ssh-keys-using-the-sha-1-signature-hash-algorithm-is-removed` if you need a detailed example of the workaround check #9640

**Reply by Jamesf86** [reply]
I test on the CLI and got an error about file permissions on ssh_known_hosts file. Could it be related to #7565? Also looks like #9691 is reporting a similar issue.  My version of ARGO-CD is v2.3.4+ac8b7df.

**Reply by 34fathombelow** [reply]
Have you checked the credentials outside of Argo?

**Reply by Jamesf86** [reply]
Stupid question should that be the argocd user account?

**Reply by Jamesf86** [reply]
okay, just switched over to using HTTPS.
